src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -488,7 +488,7 @@ await foreach (var item in enumerable)
         {
             return ReceiveWhileAsync(max, idle, filter, msgs, cancellationToken)
                 .ToListAsync(cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.